features/page-objects/WorkspacePage.js

```diff
@@ -48,6 +48,11 @@ class WorkspacePage extends Page {
     assert.equal(videoPanels.length, 1, `${videoPanels.length} was found.`)
     return await this.driver.findElement(jitsiConferenceFrame);
   }
+
+  roomCardsWhere({ accessLevel }) {
+    const selector = `.--${accessLevel.level.toLowerCase()}`
+    return this.driver.findElements(By.css(`.--${accessLevel.level.toLowerCase()}`));
```

Comment:
Not blocking, but one way we could start to distribute responsibility more broadly would be to move the responsibility for knowing how to find the accessLevel element into the other parameterTypes. I think `roomCardsWhere` still makes sense on the Workspace object, and doing the actual `driver.findElements()` within it feels good to me.

```
// in parameterTypes/workspacePage.js
class WorkspacePage {
  // ...
  roomCardsWhere({ accessLevel }) {
    return this.driver.findElements(accessLevel.selector))
  }
  // ...
}

// in parameterTypes/accessLevel.js
class AccessLevel {
  // See the MDN Getter docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get
  get selector() {
    return By.css(`.--${this.level.toLowerCase()}`)
  }
}
```

Reply:
I like this.